src/modules/brand/screens/MyBrand/MyBrand.tsx

```diff
@@ -72,6 +65,15 @@ export const MyBrand = () => {
 
   const [isBgImgModalOpened, setBgImgModalOpened] = useState(false);
 
+  const [accountInfo, setAccountInfo] = useState<IAccountInfo>();
+
+  useEffect(() => {
+    dispatch(queryAccountInfo(address))
```

Comment:
plz, use `useQuery` or `<Queries>` instead of the local state

Reply:
Since it's POST request to fetch data, the driver considers the request as mutation (not query, and useQuery doesn't work). You need to mark it explicitly as `asQuery: true` for `queryAccountInfo` meta.